packages/manager/modules/config/src/environment/environment.service.js

```diff
@@ -1,8 +1,11 @@
 import { ALLOWED_REGIONS, DEFAULT_REGION } from './environment.constants';
+import { LANGUAGES } from '../locale/locale.constants';
+import { saveUserLocale, findAvailableLocale } from '../locale';
 
 export default class EnvironmentService {
   constructor() {
     this.region = DEFAULT_REGION;
+    this.userLocale = LANGUAGES.defaultLoc;
     this.version = null;
```

Comment:
why not doing ```this.setUserLocale(detectUserLocale())```in the constructor, (since it's a singleton) so you can avoid calling this in each application ?

Reply:
Yes, could be a good thing. I we do that we could remove user locale initialization in each app (done by default).

Reply:
Done, also remove the `Environment.setUserLocale(detectUserLocale())` in each applications.